.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Everything related to the ".idea" folder should not be displayed. To avoid it let's add this folder to git ignore:

* .idea/